spec/unit/virtual_spec.rb

```diff
@@ -210,6 +210,11 @@
       Facter.fact(:virtual).value.should == "hyperv"
     end
 
+    it "should be kvm with virtio device lspci 2>/dev/null" do
+      Facter::Core::Execution.stubs(:exec).with('lspci 2>/dev/null').returns("00:03.0 Ethernet controller: Red Hat, Inc Virtio network device")
```

Comment:
Can you change this .stubs to .expects instead? It will still stub out the method. This will help ensure the test is more independent of the environment it's being run on.